apps/openmw/mwmechanics/character.cpp

```diff
@@ -254,6 +254,8 @@ void CharacterController::refreshHitRecoilAnims()
                 || mPtr.getClass().getCreatureStats(mPtr).getFatigue().getBase() == 0)
                 && mAnimation->hasAnimation("knockout"))
         {
+            mTimeToWake = time(NULL);
+            mTimeToWake += rand() % 2 + 1; // Wake up after 1 to 3 seconds
```

Comment:
This gives a time of 1-2 seconds, not 1-3. Also I'm not sure that the low-order bits of rand() are very good for generating random numbers.